.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Run tests",
+            "program": "${workspaceRoot}/node_modules/mocha/bin/mocha",
+            "args": [
+                "src/azure-openapi-validator/tests",
```

Comment:
based on our conversation I wonder if we should make this part of oav and call it with --linter ?